### DISCUSSION TITLE
怎么不写一些"润"的缺点, 只写优点, 这和观察网有啥区别了?

### DISCUSSION THREAD

**Discussion by tainzhi** [The-Run-Philosophy-Organization/run#350] [General]
"润"出去真的是比国内全方位的好吗?
当代青年被剥削严重, 承认这是反抗的一种手段。但是这个repo把国外描绘成天堂一样，真就是这样吗？

想想89年64那些学生被带头的人鼓动起来最后是什么下场？带头的人“润”到美国政治庇护，其他参与的学生全部退学，回到农村，天之骄子，成为庸人。

当然，也承认，这些为众人请命的革命者，但是普通人不明真相的群众可要想清楚了。

在这个全球比烂的时代，慎重选择。“兼听则明，偏听则暗”

**Comment by Arrnitage** [comment]
我看 润学方法论/各国选择 里写到了不同国家的坑点

**Comment by tainzhi** [comment]
好吧，我也成了没阅读全文就开喷的键盘侠。
不得不说，当前国内多年积压的矛盾在疫情的催化下越来越难掩盖了。

**Comment by tainzhi** [comment]
我还是要打开这个issue, 写的国外的缺点都是概略说过还很少, 而国内的缺点全是真实的许多案例, 这给读者造成的感情冲击明显是后者更强(作为被压迫剥削的这一代人, 难道不知道吗, 没听说过这些吗, 自己身上的重担没感受到吗?).

建议添加一些"润"的不太成功的案例, 或者"润"的过程中在步骤1, 2, 3, 4中第2步出现了差错, 后面怎么补救的措施, 毕竟"人算不如天算"

**Comment by tracyliu1** [comment]
从我个人角度陈述一点，首先润出去 一定是要能找到工作的。这对我们程序员圈子是不错的。如果是北外乔木这种，出去了不给flg写稿，就得给国内写稿。类似个别写nba一样。如果出去面临这转行，比如写代码转建筑工人、水暖工，你能学会还好，学不会一定会很难受的。
其次，还是从自身“屁股”的角度考虑，每个人的家庭条件都不一样。现在00后左右这波的实习生，很多都是大四实习然后转身去国外读研究生。不仅仅是自身条件不错，人家家里条件也足够支撑在一线城市买个可以生活的房子。相比背井离乡，图啥呢。假如你在一线城市里有两套老破小，估计你都不一定想润。
就怕你是靠工资挣钱的，然后还是普通人。

**Comment by ghost** [comment]
欢迎提交PR补充说明（

**Comment by stackunderflow111** [comment]
那没有的东西你不能硬挑是不是

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
